addons/test_mail/tests/test_mail_message.py

```diff
@@ -95,7 +95,7 @@ def test_empty_message(self):
             record._message_update_content(tracking_message, '', [])
 
     @mute_logger('odoo.models.unlink')
-    def test_mail_message_format(self):
+    def test_mail_message_format_perso(self):
```

Comment:
Note that we could add a test ensuring no "internal" information is returned when calling standard _message_format to be sure it is not added back someday by mistake. Just proposing, not saying it is really necessary (as anyway it is a private method risk is low).